doc/api/buffer.markdown

```diff
@@ -619,7 +932,7 @@ modify the length of a Buffer should therefore treat `length` as read-only and
 use [`buf.slice()`][] to create a new Buffer.
 
 ```js
-const buf = new Buffer(10);
+const buf = Buffer.allocUnsafe(10);
```

Comment:
This could be rewritten to `Buffer.from('abcdefghij', 'ascii');`
Otherwise the typo in the line below should be fixed by including the missing `i`.